### DISCUSSION TITLE
Formation/Présentation de dashlord

### DISCUSSION THREAD

**Discussion by hervechanal** [SocialGouv/dashlord#74] [General]
Bonjour,

est-il prévu une présentation de dashlord ? 

Merci par avance.

**Comment by revolunet** [comment]
Bonjour, on a une présentation de disponible [ici](https://docs.google.com/presentation/d/1dSQBmjENaVcCFrJEinFp5OzzduUSH5k_nOMROyG8wX0/edit?usp=sharing)

Vous pouvez m'envoyer un email si vous souhaitez en discuter

Cordialement
